.github/workflows/delete-packagecloud-packages.yml

```
@@ -0,0 +1,37 @@
+name: PackageCloud nightly package delete schedule
+
+env:
+  PACKAGE_CLOUD_API_TOKEN: ${{ secrets.PACKAGE_CLOUD_API_TOKEN }}
+
+on:
+  push:
+    branches:
+      - "**"
+    schedule:
+      - cron: "0 18 * * *"
```

I think once a week would be sufficient, since we are anyway deleting packages older than 10 days atm.

Also, maybe leave a link to crontab.guru so that readers can easily understand when this job gets executed: 
```suggestion
      # https://crontab.guru/#0_18_*_*_1
      - cron: "0 18 * * 1"
```

---

makes sense then